### DISCUSSION TITLE
Is there a plan to support Crossplane custom resources?

### DISCUSSION THREAD

**Discussion by Smana** [accurics/terrascan#1310] [Ideas]
Hey everyone!

I know this can be a tricky question because there are multiple providers, even for a given Cloud provider you can find several options. (e.g. provider-aws, provider-jet-aws).
I was just wondering if that was considered to scan Crossplane manifests or is there any discussion in that way?

Thanks

**Comment by gaurav-gogia** [comment]
Hi @Smana 

Crossplane is not currently in scope. However, if you wish to see this feature then please create an issue explaining your requirement.
